tests/code/test_users.py

```diff
@@ -322,3 +321,51 @@ def test_getUserBGCheckHistory():
 
 
         transaction.rollback()
+
+@pytest.mark.integration
+def test_getBannedUsers():
+    with mainDB.atomic() as transaction:
+        bannedUser = User.create(username = 'usrtst', # Test banned user
+                              firstName = 'Test',
+                              lastName = 'User',
+                              bnumber = '03522492',
+                              email = 'usert@berea.deu',
+                              isStudent = True)
+        banUser(1, User.get_by_id("usrtst"), "nope", "2022-11-29", "ramsayb2")
+        assert bannedUser in [user.user for user in getBannedUsers(1)]
+
+        unbanUser(1, User.get_by_id("usrtst"), "yep", "ramsayb2")  # Test eligible but previously banned user
+        assert bannedUser not in [user.user for user in getBannedUsers(1)]
+
+        notBannedUser = User.create(username = 'usrtst2', # Test eligible user
+                              firstName = 'Test',
+                              lastName = 'User 2',
+                              bnumber = '03522493',
+                              email = 'usert2@berea.deu',
+                              isStudent = True)
+        assert notBannedUser not in [user.user for user in getBannedUsers(1)]
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_isBannedFromEvent():
+    with mainDB.atomic() as transaction:
+        bannedUser = User.create(username = 'usrtst', # Test banned user
```

Comment:
the same comment as the previous function, the variable name bannedUser is not representative of what the code is doing.

Reply:
Changed to userToBan